change/@ni-nimble-blazor-f8fbcdc6-38cc-4c51-8016-115c43182376.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
@munteannatan I see that some changes were pushed but didn't see additional comments and there are unresolved issues so assuming it's still in work. Just FYI that @atmgrifter00 and I are OOO for the rest of this week so if comments are posted / reviews needed we won't be available until next week.